packages/object-store/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@cumulus/object-store",
+  "version": "9.0.1-alpha.0",
```

Comment:
I think we may want to explicitly specify what's included in the published package, similar to what @cumulus/checksum is doing.    

e.g. 

```
npm notice
npm notice 📦  @cumulus/object-store@9.0.1-alpha.0
npm notice === Tarball Contents ===
npm notice 986B  README.md
npm notice 805B  dist/index.d.ts
npm notice 281B  dist/index.d.ts.map
npm notice 1.4kB dist/index.js
npm notice 526B  dist/index.js.map
npm notice 1.1kB package.json
npm notice 1.2kB src/index.ts
npm notice 895B  tests/test-index.js
npm notice 91B   tsconfig.json
npm notice === Tarball Details ===
```




Right now it just defaults to `*`, which will include src, tests, etc, which will work, but we might forget to tidy it up before it goes non-alpha.